[2024-10-17T10:52:24.339+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_from_local_to_postgres.extract_data manual__2024-10-17T10:52:21.580411+00:00 [queued]>
[2024-10-17T10:52:24.352+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_from_local_to_postgres.extract_data manual__2024-10-17T10:52:21.580411+00:00 [queued]>
[2024-10-17T10:52:24.353+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-10-17T10:52:24.372+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): extract_data> on 2024-10-17 10:52:21.580411+00:00
[2024-10-17T10:52:24.380+0000] {standard_task_runner.py:60} INFO - Started process 1628 to run task
[2024-10-17T10:52:24.382+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_from_local_to_postgres', 'extract_data', 'manual__2024-10-17T10:52:21.580411+00:00', '--job-id', '678', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp6xq1yrzf']
[2024-10-17T10:52:24.385+0000] {standard_task_runner.py:88} INFO - Job 678: Subtask extract_data
[2024-10-17T10:52:24.443+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_from_local_to_postgres.extract_data manual__2024-10-17T10:52:21.580411+00:00 [running]> on host 9c093e78d1a2
[2024-10-17T10:52:24.560+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_from_local_to_postgres' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-17T10:52:21.580411+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-17T10:52:21.580411+00:00'
[2024-10-17T10:52:24.635+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-10-17T10:52:24.648+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl_from_local_to_postgres, task_id=extract_data, execution_date=20241017T105221, start_date=20241017T105224, end_date=20241017T105224
[2024-10-17T10:52:24.702+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-10-17T10:52:24.736+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
